unittests/launch_box/test_launch_box.cu

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+
+#include <gunrock/cuda/launch_box.hxx>
+
+using namespace gunrock::cuda;
+
+int main(void) {
+
+  // First template argument signifies the SM version
+  typedef launch_box_t<
+    sm_launch_params_t<86, 16, 64, 2>,
+    sm_launch_params_t<80, 16, 32, 4>,
+    sm_launch_params_t<75, 32, 64, 8>,
+    sm_launch_params_t<35, 64, 64, 16>
```

Comment:
Can you make sure there's a version where you don't specify the `smem_bytes`? For completeness of the test.